### ISSUE TITLE
Tests don't pass

### ISSUE THREAD

**Issue by davidanthoff** [WizardMac/DataRead.jl#14]
Using the latest master version of https://github.com/WizardMac/ReadStat on Windows, I get:

````
C:\Users\anthoff\.julia\v0.4\DataRead\test [bindeps ↕]> julia .\runtests.jl
WARNING: New definition
    broadcast(Function, DataArrays.PooledDataArray...) at C:\Users\anthoff\.julia\v0.4\DataArrays\src\broadcast.jl:312
is ambiguous with:
    broadcast(Any, NullableArrays.NullableArray...) at C:\Users\anthoff\.julia\v0.4\NullableArrays\src\broadcast.jl:101.
To fix, define
    broadcast(Function)
before the new definition.
Running tests:
        FAILED: read_dta.jl
LoadError: test error in expression: get(df[2,:vfloat]) == 7
NullException()
 in get at nullable.jl:30
 [inlined code] from C:\Users\anthoff\.julia\v0.4\DataRead\test\runtests.jl:13
 in anonymous at no file:0
 in include at boot.jl:261
 in include_from_node1 at loading.jl:320
 in process_options at client.jl:280
 in _start at client.jl:378
while loading C:\Users\anthoff\.julia\v0.4\DataRead\test\read_dta.jl, in expression starting on line 13ERROR: LoadError: LoadError: test error in expression: get(df[2,:vfloat]) == 7
NullException()
 in get at nullable.jl:30
 in get at nullable.jl:30
while loading C:\Users\anthoff\.julia\v0.4\DataRead\test\read_dta.jl, in expression starting on line 13
while loading C:\Users\anthoff\.julia\v0.4\DataRead\test\runtests.jl, in expression starting on line 7
````

Also pinging @matthieugomez because he added these tests.

**Comment by davidanthoff** [comment]
So the problem seems to be that all values are nulls...

**Comment by matthieugomez** [comment]
I can't really help you — I wrote most the file by trial and errors, and the tests pass on my Mac. Maybe you can find where the problem comes from by adding some @show here and there?

**Comment by davidanthoff** [comment]
Ah, and you are using the latest version of ReadStat? Then this is actually a Windows problem...

**Comment by davidanthoff** [comment]
Resolved by #15.
